src/App/Setup.php

```diff
@@ -28,6 +30,7 @@ class Setup
      */
     public function __construct(Apps $app)
     {
+        Di::getDefault()->get('log')->pushHandler(new StreamHandler('php://stdout', Logger::INFO));
```

Comment:
@rwhite27 why not save this on a property $log on the class so we don thave to call the DI all over the place